### PR TITLE
Remove projects endpoint, add project endpoint

### PR DESCRIPTION
**Pull request by ryonryon**

### CODE REVIEW

File: src/controllers/__tests__/project.test.ts [hunk 1]
```diff
@@ -1,51 +1,79 @@
...
+        url: `/project/${mockProject.id}`,
+        payload: {},
+      });
+
+      expect(res).toBe(mockProject);
     });
 
-    expect(res.statusCode).toEqual(500);
-    expect(res.body).toContain(expectedErrorMessage);
+    test.todo("Pass not existing projectId - it should successfully fetch existing project");
```
Comment (kei95) [inline src/controllers/__tests__/project.test.ts]: I'm confused by the message. Is this for the existing one? Or not the one which doesn't? Is there a reason behind that you made this test  `todo`? If you want to check it throws an error, it's doable

**Review by kei95 — APPROVED** [review]

File: tests/helper/utils.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import { Column, Project, Ticket } from "../../src/entities";
+
+export async function createTestProject(): Promise<Project> {
```
Comment (kei95) [inline tests/helper/utils.ts:3]: That's quite handy👍

File: src/controllers/__tests__/project.test.ts (hunk 1 above)
Comment (ryonryon) [reply src/controllers/__tests__/project.test.ts]: Oops, I over looked

Comment (ryonryon) [reply src/controllers/__tests__/project.test.ts]: It was todo for myself so I didn't care much grammar

**Pull request merged**